check_syntax.py

```
@@ -0,0 +1,21 @@
+import ast
+import sys
+
+try:
+    with open('d:/STUFF/WORK/N.E.K.O/main_routers/workshop_router.py', 'r', encoding='utf-8') as f:
+        content = f.read()
+    
+    ast.parse(content)
+    print("语法检查通过！")
```

Ruff (0.14.8)

String contains ambiguous `！` (FULLWIDTH EXCLAMATION MARK). Did you mean `!` (EXCLAMATION MARK)?

(RUF001)